gems/quilt_rails/lib/generators/sewing_kit/templates/sewing-kit.config.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-env node */
+
+import * as path from 'path';
+
+import {Plugins, Env} from '@shopify/sewing-kit';
+import {ReactServerPlugin} from '@shopify/react-server-webpack-plugin';
+
+const root = path.resolve(__dirname, '../');
+const tests = path.resolve(root, 'app', 'ui', 'test');
+const sourceRoot = path.resolve(root, 'app', 'ui');
+
+module.exports = function sewingKitConfig(plugins: Plugins, env: Env) {
+  return {
+    name: 'your-app-name',
+    plugins: [
+      plugins.vendors([
```

Comment:
we will eventually make this magical, but for now let's delete it anyway because the app we generate doesn't use any of these

Reply:
👍  I'll start on the auto-vendors today, so it should be ready for Hack Days.